### PR TITLE
Update template for monolithic operator pages

### PR DESCRIPTION
**Pull request by cjsha**
- MRef.extension modified to inherit contextTask info from MultiDeviceFactory for aggregate nodes as well as
- Remove rhs2116 and miniscope tutorial placeholders
- Point to later repo in source code
- Mustache templating refactored to have table and diagram partial templates to reuse reoccuring html patterns
- css modified to make headers more distinctive
- No more filtering in filter.yml - toc organized using toc.extension.js
- Not filter enums allows the cloud build to reflect my local build (which was coming out different because the api folder never clears)
- Replace the word "node" with "operator"
- node in diagram is now aligned with the input/output boxes and the grey arrows

### CODE REVIEW

File: template/ManagedReference.extension.js [hunk 1]
```diff
@@ -22,37 +22,117 @@ function replaceIObservableAndTSource(str){
...
+      //     if(model.children[j]blablablabla){
+      //       input = {
+      //         'specName': replaceIObservableAndTSource(model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters[0].type.specName[0].value),
+      //         'name': model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters[0].type.name[0].value.replaceAll(/(IObservable<)|(>)/g, ''),
+      //         'description': removeBottomMargin([ model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters[0].description, 
+      //                                             model.__global._shared['~/api/OpenEphys.Onix1.MultiDeviceFactory.yml'].children[i].syntax.parameters[0].remarks].join(''))
+      //       };
+      //     }
+      //   }
+      // }
```
Comment (bparks13) [inline template/ManagedReference.extension.js:42]: Do we need to keep these commented lines?

File: template/partials/class.tmpl.partial [hunk 2]
```diff
@@ -12,58 +12,58 @@
...
-{{>partials/class.methods}}
+{{>partials/diagram}}
 
 {{/oe.operatorType}}
 
 {{#oe.hasSubOperators}}
 
 <h2>Properties</h2>
 
+<div>{{name.0.value}} is a aggregate operator. It comprises of the following sub-operators: </div>
```
Comment (bparks13) [inline template/partials/class.tmpl.partial:46]: These are good, we should probably take this wording and apply it to the User Guide pages too

**Review by bparks13 — APPROVED** [review]
Looks good to me! Just a couple comments, let me know about the commented code and then we can merge if the checks clear.

**Comment by bparks13** [comment]
Looks like the TOC for tutorials needs to be updated and have the lines for the placeholder tutorials removed.

File: template/ManagedReference.extension.js (hunk 1 above)
Comment (cjsha) [reply template/ManagedReference.extension.js:42]: Nope. I will remove them.

Comment (cjsha) [reply template/ManagedReference.extension.js]: I removed them and also resolved some broken references in the toc due to my removal of the tutorial placeholder files (the lorem ipsum miniscope and rhs2116 pages). Another build action has initiated. Hopefully in two year when the build process completes we can merge if there are no more broken references.

**Comment by cjsha** [comment]
The ``.*DeviceFactory`` classes were preventing the linkcheck from successfully completing so I included a little if statement that said if the model in question includes ``DeviceFactory`` in its uid, toggle off the ``model.showWorkflow`` flag

**Pull request merged**